docs/transfer-flow.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Transfer FLOW
+description: How to transfer FLOW using the Flow Go SDK
+---
+
+This is an example of how to construct a FLOW token transfer transaction
+with the Flow Go SDK.
+
+Before using this example, read the [Build a Transaction guide](../building-transactions)
+for a general overview of how Flow transactions are constructed.
+
+## Cadence Script
+
+The following Cadence script will transfer FLOW tokens from a sender
+to a recipient.
+
+_Note: this transaction is only compatible with Flow Mainnet._
+
+```cadence
+import FungibleToken from 0xf233dcee88fe0abe
+import FlowToken from 0x1654653399040a61
+
+transaction(amount: UFix64, recipient: Address) {
+  let sentVault: @FungibleToken.Vault
+  prepare(signer: AuthAccount) {
+    let vaultRef = signer.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault)
+      ?? panic("failed to borrow reference to sender vault")
+
+    self.sentVault <- vaultRef.withdraw(amount: amount)
+  }
+
+  execute {
+    let receiverRef =  getAccount(recipient)
+      .getCapability(/public/flowTokenReceiver)
```

Comment:
Note that this is not yet compatible with Mainnet, only Emulator and Testnet. Maybe force-unwrap (add `!` at the end) to make this work on all networks.